apps/ios/Shared/Views/Chat/Group/GroupMemberInfoView.swift

```diff
@@ -38,8 +47,29 @@ struct GroupMemberInfoView: View {
 
                 Section("Member") {
                     infoRow("Group", groupInfo.displayName)
-                    // TODO change role
-                    // localizedInfoRow("Role", member.memberRole.text)
+
+                    HStack {
+                        if let roles = member.canChangeRoleTo(groupInfo: groupInfo) {
+                            Picker("Change role", selection: $newRole) {
```

Comment:
This picker was quite hard to find - we usually highlight interactive elements. Maybe move this under "Send direct message" and color accent? Icon could be "arrow.left.arrow.right", if picker allows icon

Reply:
Picker is now in-place...